client/core.lisp

```diff
@@ -105,11 +105,11 @@ The list is ordered alphabetically and excludes the describe-object method."
 		      (push 'integer type-list))
                      ((string-equal type "string") (push 'string type-list))
 		     ((string-equal type "boolean")
-		      (pushnew 'null type-list)
+		      (push '(eql yason:false) type-list)
 		      (push '(eql t) type-list))
```

Comment:
Maybe it we need to support `yason:true` here too?

Reply:
Also, fixtures for tests should be fixed.

Reply:
> Maybe it we need to support `yason:true` here too?

Actually, even if `yason:*parse-json-booleans-as-symbols*` is set, `cl:t` is still translated to json true. I can add it also if you want that. We could add yason:true together with cl:true so both can be used.

I personally do not like to use `yason:false` or `yason:true` symbols, but rather how e.g. [jzon does it](https://github.com/Zulu-Inuoe/jzon#type-mappings) with `cl:t` for json true, `cl:nil` for jason false, `cl:null` for json null. 

Unfortunately with yason we have to use `yason:false` for json false to enable support for json null. See [yason type mapping](http://phmarek.github.io/yason/#mapping).

Reply:
> Also, fixtures for tests should be fixed.

Forgot that. It is fixed now.